app/src/main/res/values/strings.xml

```diff
@@ -108,9 +108,9 @@
   <string name="patientedit_state" translatable="false">@string/patiententry_state</string>
   <string name="patientedit_save_patient">Save</string>
   <string name="patientedit_error_empty_fullname" translatable="false">@string/patiententry_error_empty_fullname</string>
-  <string name="patientedit_error_empty_colony_or_village" translatable="false">@string/patiententry_error_empty_colony_or_village</string>
-  <string name="patientedit_error_empty_district" translatable="false">@string/patiententry_error_empty_district</string>
-  <string name="patientedit_error_state_empty" translatable="false">@string/patiententry_error_state_empty</string>
+  <string name="patientedit_error_empty_colony_or_village" translatable="false">Required</string>
```

Comment:
Since these require translations, can we remove `translatable="false"`, please ?